data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,77 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	swizzle 6
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		"cost" 25100000
+		"shields" 37200
+		"hull" 11780
+		"required crew" 62
+		"bunks" 70
+		"mass" 725
+		"drag" 9.5
+		"heat dissipation" .4
+		"fuel capacity" 500
+		"outfit space" 872
+		"weapon capacity" 333
+		"engine capacity" 150
+		weapon
+			"blast radius" 500
+			"shield damage" 5000
+			"hull damage" 2500
+			"hit force" 3500
+	outfits
+		"Large Heat Shunt" 2
+		"Jump Drive"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Quantum Keystone"
+		"Geode Reactor"
+		"White Sun Reactor"
+		`"Bufaer" Atomic Steering`
+		Sunbeam 4
+		"Thunderhead Launcher" 4
+		"Thunderhead Missile" 160
+		"Bullfrog Anti-Missile" 2
+		"Korath Warder"
+		"Thruster (Planetary Class)"
+		"Fuel Processor" 2
+		"Wanderer Ramscoop"
+		"Pulse Rifle" 69
+		"Blade Of Might"
+	engine -45 123
+	engine 45 123
+	gun -22 -103 Sunbeam
+	gun 22 -103 Sunbeam
+	gun -34.5 -68
+	gun 34.5 -68
+	gun -48 -53.5 Sunbeam
+	gun 48 -53.5 Sunbeam
+	gun -61.5 -23
+	gun 61.5 -23
+	turret 0 5 "Korath Warder"
+	turret -55.5 74.5
+	turret 55.5 74.5
```

Comment:
Fewer guns and more turrets would be better since the Hai get most of their damage from Pulse Turrets. I recall, in testing, we found 7 guns and 5 turrets were the best option. This also leaves two spots open for reverse thrusters.

Reply:
Well I like to think that is smth that will change, for one, and it is not true for the uhai (look at the new outfits and ships PR)
The new turrets there take a lot of space and this ship design was actually intended for those and blasters